llpc/test/shaderdb/general/CsPipelineDumpTest.pipe

```diff
@@ -46,9 +46,9 @@
 
 ; Compare the disassembly to make sure all 3 Elfs have the same code. We do not expect the binaries to be bit-identical,
 ; but the text section and symbol table should be the same.
-; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d %t.orig.elf | tail -n +4 > %t.orig.s
-; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d %t/dump/PipelineCs_0x*.elf | tail -n +4 > %t.dump.s
-; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d %t.recompile.elf | tail -n +4 > %t.recompile.s
+; RUN: llvm-objdump --triple=amdgcn --mcpu=gfx900 --syms --reloc -d -j .text -j .rodata %t.orig.elf | tail -n +4 > %t.orig.s
```

Comment:
Is any change required for tests that were already using `-d` (lower case)? Doesn't that mean something like "only disassemble .text"?

Reply:
-d means disassemble (all executable sections found in the input files)
-D means disassemble-all (all sections found in the input files)
So maybe? Although using -j make sure.
Perhaps we do want to disassemble all executable sections though.